source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+
+Finally, QIIME 2 is **open-sourced and specifically designed for experienced researchers to contribute and expand the reach of their work.**
+QIIME 2 is really just a collection of wrappers called :doc:`plugins <../plugins/index>`, which can be written for any software, package, or other installable executables.
+Writing a QIIME 2 plugin for a method that you develop instantly makes it accessible and usable by thousands of users.
+
+Pro-tips for power users
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+That said, here are a few tips we've learned that should substantially improve your experience in transitioning your workflows to QIIME 2:
+
+**Pro-tip #1: QIIME 2 artifacts are just zip files**.
+If at any point you want to look at what actual files are in the ``.qza`` artifact, you can use :doc:`qiime tools export <exporting>` to extract the data file directly (which is basically just a wrapper for ``unzip``).
+Alternatively, you can also unzip your artifact directly (``unzip -k file.qza``) and look through the files in the ``data/`` folder.
+
+**Pro-tip #2: the QIIME 2 command line interface tools are slow because they have to unzip and re-zip the data contained in the artifacts each time you call them.**
+If you need to process your data more interactively, you might want to use the Python API - it is much faster since objects can be simply stored in memory.
+You can learn more about the different `QIIME 2 interfaces <https://docs.qiime2.org/2018.6/interfaces/>`__.
+
+**Pro-tip #3: to enable tab-complete in QIIME 2**, run ``source tab-qiime``.
+
+Data processing steps
+---------------------
+
+The processing steps we'll cover in this tutorial include:
+
+1. Importing data into QIIME 2
+2. Demultiplexing data (i.e. mapping each sequence to the sample it came from)
+3. Removing non-biological parts of the sequences (i.e. primers)
+4. Performing quality control and:
+
+   -  denoising sequences with DADA2 or deblur, and/or
+   -  quality filtering, length trimming, and clustering with VSEARCH or dbOTU
+
+5. Assigning taxonomy
+
+Importing data into QIIME 2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you're using QIIME 2 to process your data, the first thing you need to do is get that data into a format that QIIME 2 can understand.
+Various importing methods currently available in QIIME 2 are highlighted in the :doc:`QIIME 2 importing
+tutorial <importing>`.
+
+This step has the potential to be the most confusing part of the QIIME 2 pipeline as there are dozens of import and format types to choose from.
+To see a full list of available import/format types use: ``qiime tools import --show-importable-formats`` and ``qiime tools import --show-importable-types``
+
+If you're importing data that you've generated, you'll likely need to generate a :ref:`manifest file <manifest file>`, which is just a text file that  maps each FASTQ or FASTA file to its sample ID and direction (if applicable).
```

Comment:
If you're importing data -> If you're importing FASTQ data

Reply:
manifest files are only for fastq I believe (or so the format name would imply). @ebolyen @thermokarst is there an exception I'm not aware of?

If so, change:
that  maps each FASTQ or FASTA file to its sample ID -> that maps each FASTQ file to its sample ID

And maybe add a line about importing demultiplexed fasta data (can just point to the OTU picking tutorial, which describes that process)